### PR TITLE
Feature/har 262 navigation improvements

### PR DESCRIPTION
**Pull request by edgargressie**
Jira id: [HAR-262]
Documentation url: https://harbour.aanzee.cc/feature/HAR-262-navigation-improvements

## Checklist
- [x] Code follows the [Aan Zee conventions](https://github.com/aanzee/conventions)
- [x] SCSS imports are sorted alphabetically (when possible)

## Summary of your proposal
Add menu view and add it to the default site template.

## Proposed changes in this pull request:
* Add yml settings to editconfig
* Add menu component
* Add menu view layout
* Add `layout__section--neutral`
* Improve `$color-background-neutral`
* Merge Jekyll templates to one `default` template

### Feature
* Add menu component
* Add menu view layout
* Add `layout__section--neutral`

### Improvements
* Add yml settings to editconfig
* Improve `$color-background-neutral`
* Merge Jekyll templates to one `default` template

### CODE REVIEW

File: scss/components/menu.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
+@import 'variables/index';
+@import 'utilities/index';
+
+$menu-item-vertical-padding: $spacing * 1.5 !default;
+$menu-item-border: $border-ui-100 !default;
+$menu-padding-item-horizontal-padding: $spacing * 2 !default;
```
Comment (romypetitjean) [inline scss/components/menu.scss]: I think this should be: 'menu-item-horizontal-padding'

File: site/_includes/documentation-html.html [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {::nomarkdown}
-	</section>
+	</div>
```
Comment (romypetitjean) [inline site/_includes/documentation-html.html]: Why is this changed to div?

Comment (edgargressie) [reply site/_includes/documentation-html.html]: Since the whole documentation of the page is one section of documentation I think. When you add an example or documentation part it may be related to the content around it, so it needs to be in the same (containing) section.

File: scss/components/menu.scss (hunk 1 above)
Comment (edgargressie) [reply scss/components/menu.scss]: I think you're right, I will fix it.
